data/heb0001/heb010/__cts__.xml

```diff
@@ -0,0 +1,13 @@
+<ti:work xmlns:cpt="http://capitains.github.io/xmlns" xmlns:dc="http://purl.org/dc/elements/1.1" xmlns:dct="http://purl.org/dc/terms/" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:hebrewlit:heb0001" xml:lang="eng" urn="urn:cts:hebrewlit:heb0001.heb010">
+  <cpt:structured-metadata>
+    <dc:version></dc:version>
+  </cpt:structured-metadata>
+  <ti:title xml:lang="lat">Isaias</ti:title>
+  <cpt:structured-metadata>
+    <dct:hasVersion>urn:cts:greekLit:tlg0527.tlg048</dct:hasVersion>
+  </cpt:structured-metadata>
+  <ti:edition urn="urn:cts:hebrewlit:heb0001.heb010.1st1K-eng1" workUrn="urn:cts:hebrewlit:heb0001.heb010">
```

Comment:
This should be `<ti:translation>` instead of `<ti:edition>`. And then you will need to add an `xml:lang="eng"` attribute to this element.